src/ch01-01-installing-tools.md

```diff
@@ -1,48 +1,89 @@
-  better solution is in the works.)
+## Installing Clarinet
+
+### What is Clarinet?
+Clarinet is a Clarity runtime packaged as a command line tool, designed to facilitate smart contract understanding, development, testing and deployment. Clarinet consists of a Clarity REPL and a testing harness, which, when used together allow you to rapidly develop and test a Clarity smart contract, with the need to deploy the contract to a local devnet or testnet.
+
+Clarity is a decidable smart contract language that optimizes for predictability and security, designed for the Stacks blockchain. Smart contracts allow developers to encode essential business logic on a blockchain.
+
+### Install on macOS (Homebrew)
+
+```bash
+brew install clarinet
+```
+
+### Install on Windows
+
+The easiest way to install Clarinet on Windows is to use the MSI installer, that can be downloaded from the [releases page](https://github.com/hirosystems/clarinet/releases).
+
+Clarinet is also available on Winget, the package manager that Microsoft started including in the latest Windows updates:
+
+```powershell
+winget install clarinet
+```
+
+### Install from a pre-built binary
+
+To install Clarinet from pre-built binaries, download the latest release from the [releases page](https://github.com/hirosystems/clarinet/releases).
+Unzip the binary, then copy it to a location that is already in your path, such as `/usr/local/bin`.
+
+```sh
+# note: you can change the v0.27.0 with version that are available in the releases page.
+wget -nv https://github.com/hirosystems/clarinet/releases/download/v0.27.0/clarinet-linux-x64-glibc.tar.gz -O clarinet-linux-x64.tar.gz
+tar -xf clarinet-linux-x64.tar.gz
+chmod +x ./clarinet
+mv ./clarinet /usr/local/bin
+```
+
+On MacOS, you may get security errors when trying to run the pre-compiled binary. You can resolve the security warning
+with with command
+
+```sh
+xattr -d com.apple.quarantine /path/to/downloaded/clarinet/binary
+```
+
+### Install from source using Cargo
+
+#### Prerequisites
+
+[Install Rust](https://www.rust-lang.org/tools/install) for access to `cargo`, the Rust package manager.
+
+On Debian and Ubuntu-based distributions, please install the following packages before building Clarinet.
+
+```bash
+sudo apt install build-essential pkg-config libssl-dev
+```
+
+#### Build Clarinet
+
+You can build Clarinet from source using Cargo with the following commands:
+
+```bash
+git clone https://github.com/hirosystems/clarinet.git --recursive
+cd clarinet
+cargo clarinet-install
+```
+
+By default, you will be in our development branch, `develop`, with code that has not been released yet. If you plan to submit any changes to the code, then this is the right branch for you. If you just want the latest stable version, switch to the main branch:
+
+```bash
+git checkout main
+```
+
+If you have previously checked out the source, ensure you have the latest code (including submodules) before building using:
+
+```
```

Comment:
Missing `bash` language specifier for the code block. Leads to a render issue with the current build script. (See https://deploy-preview-15--claritybook.netlify.app/ch01-01-installing-tools.html#build-clarinet.)